src/decision/dMCDA.jl

```diff
@@ -771,7 +800,15 @@ Here, `max_cover` represents the max. carrying capacity for each site (the `k` v
 - `available_space` : vector/matrix : space available at each site (`k` value)
 - `depth` : vector of site ids found to be within desired depth range
 """
-function unguided_site_selection(prefseedsites, prefshadesites, seed_years, shade_years, n_site_int, available_space, depth)
+function unguided_site_selection(
+    prefseedsites,
+    prefshadesites,
```

Comment:
Missing docstring for `prefshadesites`.

While you're here, you might as well standardize variable names to snake case.